### ISSUE TITLE
Lots of Title ID's not found in titledb causing lots of lag? Any solution for this??

### ISSUE THREAD

**Issue by brendongl** [a1ex4/ownfoil#158]
[2024-12-18 00:25:05.737] ERROR (titles) Title ID not found in titledb: 01005EF003FF3002
[2024-12-18 00:25:05.940] ERROR (titles) Title ID not found in titledb: 0100AC601A844000
[2024-12-18 00:25:06.266] ERROR (titles) Title ID not found in titledb: 010035901701B003
[2024-12-18 00:25:06.471] ERROR (titles) Title ID not found in titledb: 01002E80168F5001
[2024-12-18 00:25:11.721] ERROR (titles) Title ID not found in titledb: 01008BA01A3BF007
[2024-12-18 00:25:11.736] ERROR (titles) Title ID not found in titledb: 01008BA01A3BF006
[2024-12-18 00:25:13.130] ERROR (titles) Title ID not found in titledb: 01006FB00EBE1010
[2024-12-18 00:25:13.145] ERROR (titles) Title ID not found in titledb: 01006FB00EBE100F
[2024-12-18 00:25:13.384] ERROR (titles) Title ID not found in titledb: 01008BA01A3BF006
[2024-12-18 00:25:17.448] ERROR (titles) Title ID not found in titledb: 0100E0901E2AB009
[2024-12-18 00:25:17.509] ERROR (titles) Title ID not found in titledb: 0100E0901E2AB007
[2024-12-18 00:25:17.913] ERROR (titles) Title ID not found in titledb: 01777FAB1777F000
[2024-12-18 00:25:18.168] ERROR (titles) Title ID not found in titledb: 010013001BD63004
[2024-12-18 00:25:18.878] ERROR (titles) Title ID not found in titledb: 010025C0145D5099
[2024-12-18 00:25:19.401] ERROR (titles) Title ID not found in titledb: 01003310159F9025
[2024-12-18 00:25:19.816] ERROR (titles) Title ID not found in titledb: 01006B601E6DC000
[2024-12-18 00:25:20.676] ERROR (titles) Title ID not found in titledb: 01006C2012714000
[2024-12-18 00:25:24.151] ERROR (titles) Title ID not found in titledb: 0100D74015CF0000
[2024-12-18 00:25:25.137] ERROR (titles) Title ID not found in titledb: 010078D000F8900A
[2024-12-18 00:25:27.518] ERROR (titles) Title ID not found in titledb: 01006DD01EB64000
[2024-12-18 00:25:28.866] ERROR (titles) Title ID not found in titledb: 010050801E0CC000
[2024-12-18 00:25:34.374] ERROR (titles) Title ID not found in titledb: 010069B01B18F007
[2024-12-18 00:25:38.120] ERROR (titles) Title ID not found in titledb: 01000F700DECF099

I know the titledb isnt managed by you. But can we add some setting where if it scans it once and the title is not in titledb, to ignore the file in future scans (also a setting which allows us to skip the ignore to check again)

I can't figure out which games these are either because my games don't have filenames, so it would take me hours of manual work googling the titleid then scouring the files.

Thanks again for your amazing work btw!

**Comment by a1ex4** [comment]
Yes, the plan is to mark all the files which cannot be correctly identified as such, stop trying to scan/identify them and have a view in the UI to troubleshoot those file.
